.github/ISSUE_TEMPLATE/issue_bug_template.md

```diff
@@ -0,0 +1,33 @@
+---
+name: ‼ Bug 🐛
+about: Describe the bug in Provider Implementation
```

Comment:
```suggestion
about: Describe the bug to be fixed
```

(the PR is against lead repository, not Provider Implementation ... not sure if all section below will apply)

Reply:
As Bug template was intended for Provider Implementation repositories, I removed it from here.
For "bugs" in API specifications the Correction template can be used.